content/00_images_are_arrays.md

```diff
@@ -0,0 +1,366 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.3
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+```{code-cell} ipython3
+:tags: [remove-input, remove-output]
+
+%matplotlib inline
+%config InlineBackend.figure_format = 'retina'
+```
+
+# Images are numpy arrays
+
++++
+
+Images are represented in ``scikit-image`` using standard ``numpy`` arrays.  This allows maximum inter-operability with other libraries in the scientific Python ecosystem, such as ``matplotlib`` and ``scipy``.
+
+Let's see how to build a grayscale image as a 2D array:
+
+```{code-cell} ipython3
+import numpy as np
+from matplotlib import pyplot as plt
+
+random_image = np.random.random([500, 500])
+
+plt.imshow(random_image, cmap='gray')
+plt.colorbar();
+```
+
+The same holds for "real-world" images:
+
+```{code-cell} ipython3
+from skimage import data
+
+coins = data.coins()
+
+print('Type:', type(coins))
+print('dtype:', coins.dtype)
+print('shape:', coins.shape)
+
+plt.imshow(coins, cmap='gray');
+```
+
+A color image is a 3D array, where the last dimension has size 3 and represents the red, green, and blue channels:
+
+```{code-cell} ipython3
+cat = data.chelsea()
+print("Shape:", cat.shape)
+print("Values min/max:", cat.min(), cat.max())
+
+plt.imshow(cat);
+```
+
+These are *just NumPy arrays*. E.g., we can make a red square by using standard array slicing and manipulation:
+
+```{code-cell} ipython3
+cat[10:110, 10:110, :] = [255, 0, 0]  # [red, green, blue]
+plt.imshow(cat);
+```
+
+Images can also include transparent regions by adding a 4th dimension, called an *alpha layer*.
+
++++
+
+### Other shapes, and their meanings
+
+|Image type|Coordinates|
+|:---|:---|
+|2D grayscale|(row, column)|
+|2D multichannel|(row, column, channel)|
+|3D grayscale (or volumetric) |(plane, row, column)|
+|3D multichannel|(plane, row, column, channel)|
+
++++
+
+## Displaying images using matplotlib
+
+```{code-cell} ipython3
```

Comment:
Can these just say "python"?

Reply:
I'll test it; I converted these automatically.